docs/configuration.md

```diff
@@ -94,6 +96,22 @@ The following variables are set.
 | OPENSTACK_CLOUD_PROVIDER_CONF_B64 | The content of [cloud.conf](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf) which is used by OpenStack cloud provider |
 | OPENSTACK_CLOUD_CACERT_B64 | (Optional) The content of your custom CA file which can be specified in your clouds.yaml by `ca-file` |
 
+[Application Credentials](https://docs.openstack.org/keystone/latest/user/application_credentials.html) is used for applications to authenticate to keystone. This way, the user’s password is not embedded in the application’s configuration, which is especially important for users whose identities are managed by an external system such as LDAP or a single-signon system.
```

Comment:
We can delegate the explanation of Application Credential to the link. Please remove L99.

Reply:
ok